libs/shared/contentful/src/lib/factories.ts

```diff
@@ -50,6 +55,54 @@ export const EligibilityContentByPlanIdsQueryFactory = (
   };
 };
 
+export const EligibilityOfferingResultFactory = (
+  override?: Partial<EligibilityOfferingResult>,
+  subGroupCollectionExtension?: EligibilitySubgroupResult[],
+  subGroupOfferingCollectionExtension?: EligibilitySubgroupOfferingResult[]
```

Comment:
This is non-blocking, but I've been recommending people break their factories into smaller factories that consume each other in this scenario.

If you'd like to keep it as-is, that's fine. I'd just ask that we change the name of these args to end in `Override` rather than `Extension`.

Reply:
It's not actually overriding existing values, its extending. That was the main reason I changed the name.

Reply:
Oh, missed that.

Reply:
I'd still recommend the "break it into smaller factories" thing, but understood on the variable names. Still non-blocking.